### ISSUE TITLE
fatal error while scanning a php file

### ISSUE THREAD

**Issue by mjrider** [php-gettext/PHP-Scanner#1]
I'm getting the following error:
```
PHP Fatal error:  Uncaught Error: Call to undefined method PhpParser\Node\Expr\Variable::getLast() in .../vendor/gettext/php-scanner/src/PhpNodeVisitor.php:31
```

the line of code triggering the error is:
```php
    $phpScanner->scanFile($file->getRelativePathname());
```

the contents of the file which would still trigger the error
```php
<?php

class AuthMiddleware
{
    public function test()
    {
       return __('Accessing the Backoffice is not allowed at this time.');
    }
}
```

Versions:
```
gettext/gettext                     v5.0.0             PHP gettext manager
gettext/languages                   2.5.0              gettext languages with plural rules
gettext/php-scanner                 v1.0.0             PHP scanner for gettext
```

**Comment by oscarotero** [comment]
Hi, @mjrider 
Sorry for the late response, for some reason, I wasn't waching this repo, so didn't get any notification.

I just commit your code but couldn't reproduce this error. https://github.com/php-gettext/PHP-Scanner/commit/f9f0bedd2eefc06c34a9b5e6d2ce4b3d6324a7eb
Could I get more info? Maybe by creating a test case reproducing the error?

**Comment by mjrider** [comment]
i reduced the script to trigger the error to
```php
<?php
require_once __DIR__ . '/../vendor/autoload.php';

use Gettext\Scanner\PhpScanner;
use Gettext\Translations;

$phpScanner = new PhpScanner(
    Translations::create()
);

$phpScanner->scanFile('app/Lib/Helpers.php');
$phpScanner->scanFile('app/Middlewares/AuthMiddleware.php');
```

first file doesn't have translations, works fine
second file has 1 translation, and fails

But i still cant get it to fail in the php-scanner unti tests

**Comment by mjrider** [comment]
it seems that i didn't produce the correct reproduction scenario.

see https://github.com/php-gettext/PHP-Scanner/compare/master...mjrider:master for the change in the testcase

**Comment by oscarotero** [comment]
Fixed in v1.0.1
Thanks for reporting!
